server/app/routes/main.py

```
@@ -37,17 +42,37 @@ def index():
         url=config.URL,
+        max_courses_taken=config.MAX_COURSES_TAKEN,
     )
 
 
 @main_bp.route("/unlocks", methods=["POST"])
 def unlocks_redirect():
-    raw = request.form.get("code", "")
-    sem = request.form.get("semester", current_app.config["DEFAULT_SEMESTER"])
-    base = normalise(raw)
-    if not base:
+    raw = request.form.get(f"tentative-code", "")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)